Maquetacion/index.html

```diff
@@ -10,12 +10,82 @@
   </head>
   <body>
     <h1 class="title">Front-End Tools</h1>
-    <div class="tech">
-      <img class="logo" src="assets/react_icon.svg" alt="React">
-      <div>
-        <h2 class="text title-card">ReactJS</h2>
-        <p class="text age">Release: 2013</p>
-        <p class="text story">React is a delarative, component based JavaScript library for bouilding user interfaces</p>
+    <div class="front">
```

Comment:
Try using more declarative names for the classes names. This div could use the class name "section" since it's used in both "sections" of the website. (Front-end tools and Back-end tools)